### ISSUE TITLE
deConz setup is unclear

### ISSUE THREAD

**Issue by skipishere** [home-assistant/home-assistant.github.io#18673]
### Feedback

The instructions for getting deConz to work with Home Assistant appears to be missing some information on how to get up and running.

In my example, my Conbee II is connected into my Home Assistant Raspberry Pi which uses the Home Assistant OS.
The documentation (and the below message) don't tell you how you're meant to get to the gateway to do the next steps
![image](https://user-images.githubusercontent.com/26724777/127737617-3dc3de6f-2fcf-4e56-ad26-c780372d90d9.png)

### URL

https://www.home-assistant.io/integrations/deconz/

### Version

2021.7.4

**Comment by Kane610** [comment]
This knowledge is expected pre-knowledge since the user needs to set up deconz prior to setting up the integration. What kind of information are you suggesting would help here?

**Comment by skipishere** [comment]
@Kane610 Given I really didn't get it to work fully, I'm not sure what steps are needed/missing.

From further investigating I assume I needed to install the deCONZ addon (from Supervisor) as well as the deCONZ in the integrations? If that's the case the documentation should really explain that at minimum or at least have some link/steps/further tutorial link. I think expecting it to be pre-knowledge really hinders new starters using it (I started with ZHA and ultimately gave up and went back to ZHA as it was a far easier onboarding experience)

**Comment by frenck** [comment]
You need to have deCONZ to be able to integrate with deCONZ. Just like you need have Philips Hue in order to integrate with Philips Hue.

I'm not sure how that could be confusing?

**Comment by skipishere** [comment]
But in this instance you need the stick (in my case a Conbee II), deCONZ in integrations (which is what the article is talking about) but also deCONZ in Supervisor (which feels like that part that is missing/lacking) which differs from other integrations when being added (so more a point of confusion than anything else)

If I'd got it fully working I'd have updated the docs with changes, but as I ended up aborting and moving back to Conbee II + ZHA integration (and nothing in supervisor) I'm unsure with what needs to be altered. (Hope that's not too confusing, really want to make the docs more new user friendly as HA is a great system 🙂)

**Comment by frenck** [comment]
> But in this instance you need the stick (in my case a Conbee II), deCONZ in integrations (which is what the article is talking about) but also deCONZ in Supervisor (which feels like that part that is missing/lacking) which differs from other integrations when being added (so more a point of confusion than anything else)

Sorry, I really can't get along with these statements. For hue you need hardware as well, the hub. 
The integration is what you are looking at in Home Assistant (all are integrations...?).

As for rest of the text, that depends on how you want to run deCONZ. There are many ways, see the website of the manufacturer for that information. For example, the deCONZ add-ons (and thus the supervisor) is not needed. That is a choice.

We don't have a guide on how to install your Hue hub either. This is not our scope.

**Comment by skipishere** [comment]
I don't think my point is coming across clearly.

When I was using ZHA - I connected the Conbee II stick, added the integration for ZHA and then selected the stick in the options and was good to go.

vs

When trying to use the deConz I connected the stick and then added the integration which presented me with the dialog shown (in my first message). Using Home Assistant OS there is no way I can find the IP address of the stick plugged physically into the USB of the Home Assistant machine (Raspberry Pi) and without adding the Supervisor addon there is also no way for me to click the 'authenticate app' button.

I hope this makes it a bit clearer what I'm trying to explain.
